docs/source/reference/northstar/chathooks.rst

```diff
@@ -112,6 +114,10 @@ players, they only display them locally.
             wait 1
             Chat_GameWriteLine("BIOMETRIC MONITORING SYSTEMS-")
             ActivateBiometricMonitoringSystems()
+            Chat_GameWrite("ACTIVATED")
+            wait 1
+            Chat_GameWriteLine("HAVE A VERY SAFE DAY.")
```

Comment:
Not sure if this is necessary, but it might be nice to include a small comment about the difference between these two functions